test/integration/wallet.test.js

```diff
@@ -0,0 +1,133 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+const authService = require('../../services/authService')
+
+const addUser = require('../utils/addUser')
+const cleanDb = require('../utils/cleanDb')
+const usersUtils = require('../../utils/users')
+
+const userData = require('../fixtures/user/user')()
+const { walletBodyKeys, walletKeys, walletDataKeys } = require('../fixtures/wallet/wallet')
+
+const defaultUser = userData[0]
+const newUser = userData[3]
+const superUser = userData[4]
+
+const config = require('config')
+const cookieName = config.get('userToken.cookieName')
+
+chai.use(chaiHttp)
+
+describe('Wallet', function () {
+  let authToken
+  let userId
+  let userName
+
+  beforeEach(async function () {
+    userId = await addUser(defaultUser)
+    authToken = authService.generateAuthToken({ userId })
+    userName = await usersUtils.getUsername(userId)
+  })
+
+  afterEach(async function () {
+    await cleanDb()
+  })
+
+  describe('Check /wallet', function () {
```

Comment:
Let's name this as `GET /wallet` maybe?
Once we run the tests, it would make more sense then 😉

Reply:
done